src/components/ProductDetails/ProductDetails.jsx

```diff
@@ -40,7 +41,10 @@ class ProductDetails extends React.Component {
       data.data.results.forEach((result) => {
         if (result["default?"]) {
           // console.log('enter default')
-          this.setState({currentStyle: result})
+          this.setState({currentStyle: result});

```

Comment:
updates state on componentDidUpdate, if there are no default style it sets the default style to the first style in the styles data array.